service/src/test/kotlin/app/cash/backfila/service/BackfillRunnerTest.kt

```diff
@@ -355,7 +329,7 @@ class BackfillRunnerTest {
       }
 
       // Give enough time for the once-per-second check to cancel the jobs.
-      delay(1100)
+      delay(EXTEND_LEASE_PERIOD.toMillis())
```

Comment:
plus a little bit?

Reply:
Or perhaps half the lease period?

Reply:
doesn't have to be plus a bit, this consistently works